add blog post on EUM'26 - day 1 (#15)
easybuilders/easybuild-blog: docs/posts/2026/04/eum26-day1.md

<figure markdown="span">
![](eum26_group_picture.webp){width=100%}
</figure>

The EasyBuild community (and beyond) gathered again to learn more about recent developments,
letting HPC sites that use EasyBuild showcase how they are doing so, attend presentations on
projects (loosely) related to EasyBuild, and discuss the future of EasyBuild,
including the (potential) impact of LLMs.

The event was sponsored by [Centro Nacional de Computação Avançada (the Portuguese National Center
for Advanced Computing)](https://acnca.pt/?lang=en), [INESC TEC](https://www.inesctec.pt/en),
[Fujitsu](https://global.fujitsu/), [Do IT Now](https://www.doitnowgroup.com/),
[Inuits](https://inuits.eu/), [Microsoft Azure](https://azure.microsoft.com/en-us),
and [NVIDIA](https://www.nvidia.com/).

This blog post covers the first day of the 3-day event, see also [day 2](eum26-day2.md) and [day 3](eum26-day3.md).

<!-- more -->

## Kickstart with social event

The 11th EasyBuild User Meeting (EUM) started off a bit atypically: with a tour of the local EuroHPC supercomputer
named [*Deucalion*](https://www.macc.fccn.pt/resources#deucalion), followed by a guided tour of Guimarães,

> the host city for the event and also the birth place of Portugal.

Reviewer: ```suggestion
the host city for the event and also the birthplace of Portugal.
```

Social activities have been part of the EUM program before, especially if you consider the group dinners,
and the "team building"-like side steps we have done in recent years. See for example the karting race that was
organised to celebrate the [10th EasyBuild User Meeting last year](../../2025/03/eum25.md).

Nevertheless, attendees took the opportunity to meet up with people they haven't seen in a while, or perhaps have

> only met online until then. The custom professsionally designed badges that everyone was invited to wear 

Reviewer: ```suggestion
only met online until then. The custom professionally designed badges that everyone was invited to wear 
```
Come on codespell, get in the game

Author: Interesting that it missed this, nice catch!

were a great conversation starter, since they also feature GitHub handles. In some cases, even the band T-shirt
that some people happened to wear triggered people to talk to each other. :wink:

<figure markdown="span" style="display:flex; gap:0; justify-content:center;">
![](eum26_registration.webp){width=50%}
![](eum26_badge.webp){width=50%}
</figure>

### Visit of Deucalion

Bernardo from the Deucalion team kickstarted the visit by explaining a couple of things outside first.

<figure markdown="span">
![](eum26_deucalion-visit-outside.webp){width=100%}
</figure>

In small self-organised groups, we walked through the noisy and (surprisingly) chilly room in which Deucalion
was humming away running scientific calculations for researchers from Portugal and all across Europe.

<figure markdown="span">
![](eum26_deucalion-visit-group-pic-1.webp){width=100%}
</figure>

<figure markdown="span">
![](eum26_deucalion-visit-group-pic-2.webp){width=100%}
</figure>

We got to see the relatively new artwork and custom logo of Deucalion up close,

> and many agreed that is was really well done.

Reviewer: ```suggestion
and many agreed that this was really well done.
```

*([slides](https://users.ugent.be/~kehoste/eum26/eum26_002_governance.pdf) - [recording](https://www.youtube.com/watch?v=rXuwGFUZqFk&list=PLhnGtSmEGEQjTW4HoDDNnjBGUf4v0RgJw&index=2&pp=iAQB0gcJCdQKAYcqIYzv))*

## HPSF

Directly tying in with the discussion on governance, the next talk introduced the
[*High Performance Software Foundation (HPSF)*](https://hpsf.io).

Massimiliano explained what HPSF is all about, how HPC providers or sites can become a member,
why projects should consider joining HPSF, and what the process entails.

<figure markdown="span" style="display:flex; gap:0; justify-content:center;">
![](eum26_hpsf-screenshot1.webp){width=50%}
![](eum26_hpsf-screenshot2.webp){width=50%}
</figure>

Both EasyBuild and EESSI have expressed interest in joining HPSF, so this presentation was particularly
interesting to attendees.

*([slides](https://users.ugent.be/~kehoste/eum26/eum26_003_HPSF.pdf) - [recording](https://www.youtube.com/watch?v=2BeWQquZbR8&list=PLhnGtSmEGEQjTW4HoDDNnjBGUf4v0RgJw&index=4&pp=iAQB))*

## EasyBuild site talks, part 1

To wrap up the first day, 3 (actually 4) people gave a short EasyBuild site talk, another tradition that has grown 

> a the EasyBuild User Meeting throughout the years.

Reviewer: ```suggestion
at the EasyBuild User Meeting throughout the years.
```
